earth_enterprise/src/fusion/portableglobe/rewritedbroot.cpp

```diff
@@ -162,6 +162,19 @@ void ReplaceSearchServer(const std::string &search_service,
   AddStringToProto(url, search_server->mutable_url());
 }
 
+/**
+ * Removes the reference to the server that the dbroot references for 
+ * historical imagery.
+ * @param dbroot geProtoDbroot to be modified
+ */
+void DisableTimeMachine(geProtoDbroot *dbroot) {
+  assert(dbroot); 
+  keyhole::dbroot::EndSnippetProto* end_snippet =
+      dbroot->mutable_end_snippet();
+  keyhole::dbroot::TimeMachineOptionsProto* time_machine_options = end_snippet->mutable_time_machine_options(); 
```

Comment:
I don't know how strict we are at code formatting but I figured I'd go ahead and point out the length of this new line is probably too long.  I'm also using this to try out the "suggestion" feature.  :-)

```suggestion
  keyhole::dbroot::TimeMachineOptionsProto* time_machine_options =
      end_snippet->mutable_time_machine_options(); 
```